### ISSUE TITLE
Download Closed Captions of Twitch Stream VODS

### ISSUE THREAD

**Issue by jovaig** [lay295/TwitchDownloader#259]
Was wondering if you are able to create a feature that allows us to download the closed captions (either in .srt or .txt format) of Twitch Stream VODS. We already chat transcript downloading, what about the transcript of the actual stream?

**Comment by lay295** [comment]
Does Twitch provide closed captions on VODs? When I load up a VOD I don't see any closed caption button. Could you link one if so?

**Comment by jovaig** [comment]
> Does Twitch provide closed captions on VODs? When I load up a VOD I don't see any closed caption button. Could you link one if so?

Yeah some streamers can use a Twitch extension to get closed captions on their streams: https://stream-cc.gooseman.codes/

Here is an example of a VOD with closed captions: https://www.twitch.tv/videos/1161062053?t=0h29m45s

It might take a few second for the CC button to show up or you can press c
